pkg/network/driver_interface.go

```diff
@@ -181,13 +184,25 @@ func (di *DriverInterface) GetStats() (map[string]interface{}, error) {
 		return nil, err
 	}
 	totalFlows := atomic.LoadInt64(&di.totalFlows)
+	openFlows := atomic.LoadInt64(&di.openFlows)
+	closedFlows := atomic.LoadInt64(&di.closedFlows)
+	moreDataErrors := atomic.SwapInt64(&di.moreDataErrors, 0)
```

Comment:
Why is this stat being reset and the others not?

Reply:
These were largely for testing and it was more useful to not have it be a monotonic count. We didn't have a need for the others being reset, so we didn't do it. 

Now that you point it out, though, I think it does make sense to have open/closed flows just be for a given interval and keep the total monotonically increasing. 

Going to ping @derekwbrown for his thoughts as well.